Misc/NEWS.d/next/Windows/2024-05-30-17-39-25.gh-issue-119679.mZC87w.rst

```diff
@@ -0,0 +1 @@
+Ensures correct import libraries are included in Windows installs.
```

Comment:
if my understanding is correct, this changes what libraries are included only when opting in to the free-threaded binaries, right?

```suggestion
Ensures correct import libraries are included in Windows installs when installing free-threaded binaries.
```

Reply:
It also changes the ones included in the Nuget package or anything you might create using the `PC/layout` script, but it was noticed because of the free threaded package. It's only a fix for a beta, so I'm not too concerned about people noticing and needing to check the release notes.